src/test/resources/features/login.feature

```diff
@@ -13,3 +13,13 @@ Feature: Login
     And User clicks login button
     Then User is not logged in
 
+  Scenario: User tries different invalid credentials
+    Given User is on the login page
+    When User tries to login with invalid credentials
```

Comment:
you need to validate that user is not logged in with every new entry:
 When User tries to login with invalid credentials
      | username            | password            |
      | bad username        | bad password        |
    Then User is not logged in
    When User tries to login with invalid credentials
      | username            | password            |
      | wrongUsername123    | SuperWrongPassword! |
    Then User is not logged in
    When User tries to login with invalid credentials
      | username            | password            |
      | some other username | incorrect Password15|
    Then User is not logged in

Reply:
fixed